core/arch/arm/plat-k3/drivers/ti_sci.h

```diff
@@ -124,6 +124,43 @@ int ti_sci_get_dkek(uint8_t sa2ul_instance,
 		    const char *context, const char *label,
 		    uint8_t dkek[SA2UL_DKEK_KEY_LEN]);
 
+/**
+ * ti_sci_read_otp_mmr() - Get the Extended OTP
+ * @mmr_idx:	        32-bit MMR index
+ * @val:		Value of the 32-bit MMR
+ *
+ * Reads the extended OTP bits from efuse
+ *
+ * Return: 0 if all goes well, else appropriate error message
+ */
```

Comment:
Inline description comment should be only at one place, either in the header file where declared or in the source file where defined, but preferrably not both.